src/Components/Layout/LayoutComponents/Footer.css

```diff
@@ -0,0 +1,16 @@
+.footer-bg {
+  display: flex;
```

Comment:
- [ ] You have used display: flex more than 10 times, which falls under the category of bad practice since you are repeating yourself a lot. KIndly create a class with display: flex inside and reuse it whenever necessary